psalm.xml

```diff
@@ -41,17 +41,21 @@
         </NullArgument>
         <PossiblyNullReference>
             <errorLevel type="suppress">
+                <!-- ReflectionProperty::getType() can return null, but there is ReflectionProperty::hasType()
```

Comment:
This has been fixed in Psalm recently:
https://github.com/vimeo/psalm/issues/5258
Maybe Psalm can be upgraded?

Reply:
Here is `ReflectionProperty` (instead of `ReflectionParameter`) I've created https://github.com/vimeo/psalm/pull/5584 based on https://github.com/vimeo/psalm/issues/5258, but after reading it again, IIUC, it won't work because `ReflectionProperty` is not immutable.

Reply:
Oh,  my mistake then :)